platform/location_service/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+project(location_service)
```

Comment:
nit
```suggestion
project(location_service LANGUAGES CXX OBJCXX)
```

Reply:
`CXX` is enabled by default and we should not repeat ourselves.
Adding `OBJCXX` would completely break existing Linux builds, as adding this would force the Linux systems to also install an `objective-c++` compiler all of a sudden, for no actual reason, as it is not needed for a Linux build at all. So I don't think that these changes are in our interest.